src/service/okapi.service.ts

```diff
@@ -27,18 +27,19 @@ class OkapiService extends RestService {
           'Accept': 'application/json'
         }
       }, (error: any, response: any, body: any) => {
-        if (response && response.statusCode >= 200 && response.statusCode <= 299) {
+        if (!!response && response?.statusCode >= 200 && response?.statusCode <= 299) {
```

Comment:
Remove unnecessary conditions.

```suggestion
        if (!!response && response.statusCode >= 200 && response.statusCode <= 299) {
```

Reply:
These are required as the existence of `response` does not mean `statusCode` is a present property.

Reply:
This is still a copy and paste of an incorrect implementation that includes redundant conditions.